### ISSUE TITLE
Looking for cones in SRS Arc plans and checking if Jaws are set to 4x4 cm. 

### ISSUE THREAD

**Issue by kkaan** [CCCC-stygit/MegaCheck#2]
SRS plans are no longer using cones and jaws can be of any size above a 3x3 (but this 3x3 aperture can if off-axis so jaw position cannot be used).
